src/data-browser/graphs/quarterly/index.jsx

```diff
@@ -1,22 +1,32 @@
-import React, { useState } from 'react'
-import { GraphsHeader } from './GraphsHeader'
-import { HomeLink } from '../../HomeLink'
-import { SectionFAQ } from './SectionFAQ.jsx'
-import { SectionFilerInfo } from './SectionFilerInfo'
-import { SectionGraphs } from './SectionGraphs'
-import { SectionSelector } from '../SectionSelector'
-import Error from '../../../common/Error'
-import '../graphs.css'
+import React, { useEffect, useState } from "react"
+import { GraphsHeader } from "./GraphsHeader"
+import { HomeLink } from "../../HomeLink"
+import { SectionFAQ } from "./SectionFAQ.jsx"
+import { SectionFilerInfo } from "./SectionFilerInfo"
+import { SectionGraphs } from "./SectionGraphs"
+import { SectionSelector } from "../SectionSelector"
+import { Switch, Route } from "react-router-dom"
+import Error from "../../../common/Error"
+import "../graphs.css"
 
-const SectionOptions = ['Graphs', 'Filer Info', 'FAQ']
+const SectionOptions = ["Graphs", "Filer", "FAQ"]
```

Comment:
I still prefer `Filer Info` over `Filer` for the nav-link label.  I think it's more descriptive of what the user will find in that section, even it doesn't exactly match the URL endpoint.

Reply:
The `SectionOptions` array controls the label name and what appears in the URL. 

If we revert back to `Filer Info` the URL will say `info/filer-info`. Are you okay with it saying this?

Reply:
Could you extend the logic to only use the first word of the label as the URL reference? (split at space, take first word, remove any non-alphabetic characters).

Or create a map from `label => url`?

Reply:
This has been fixed by using an object instead of a string.

`{label: 'Filers Info', url: '/filers'}`.